mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -243,15 +243,17 @@ public void hideKeyboard() {
         InputMethodManager imm
                 = (InputMethodManager) getSystemService(Activity.INPUT_METHOD_SERVICE);
         View view = getCurrentFocus();
-        if (view == null && imm != null) {
-            view = new View(this);
+        if (imm != null) {
```

Comment:
If this is the solution, then you can simply put `imm != null` before `&&` and you will be fine.